dht-node/src/config/index.ts

```diff
@@ -28,6 +28,12 @@ const database = {
     process.env.TYPEORM_LOGGING_RELATIVE_PATH || 'typeorm.dht-node.log',
 }
 
+const community = {
+  COMMUNITY_NAME: process.env.COMMUNITY_NAME || 'Gradido Entwicklung',
+  COMMUNITY_DESCRIPTION:
+    process.env.COMMUNITY_DESCRIPTION || 'Gradido-Community einer lokalen Entwicklungsumgebung.',
+}
+
```

Comment:
please separate this in its own PR. This is its own thing?!

Reply:
No this is part of the business community entry, which is the main part of this PR